api/sheet.go

```diff
@@ -120,6 +182,9 @@ type SheetCreate struct {
 	Name       string          `jsonapi:"attr,name"`
 	Statement  string          `jsonapi:"attr,statement"`
 	Visibility SheetVisibility `jsonapi:"attr,visibility"`
+	Source     SheetSource
+	Type       SheetType
+	Payload    string
```

Comment:
Even though we store in json format, shall we use declarative fields in the API to save additional conversions from the client side? We may play with json SQL queries.

https://www.postgresql.org/docs/9.3/functions-json.html

Reply:
@tianzhou WDYT?

Reply:
Agreed.

Reply:
Payload tags updated. And we can use it to save tab info for SQL Editor in next. @xiaoluoboding